sklearn/linear_model/tests/test_ridge.py

```diff
@@ -726,6 +734,20 @@ def _test_ridge_classifiers(filter_):
     assert np.mean(y_iris == y_pred) >= 0.8
 
 
+@pytest.mark.parametrize("scoring", [None, "accuracy", _accuracy_callable])
+@pytest.mark.parametrize("cv", [None, KFold(5)])
+@pytest.mark.parametrize("filter_", [DENSE_FILTER, SPARSE_FILTER])
```

Comment:
I would test also with a dummy callable that return let's say 0.42 all the time and check that the score I obtain is actually 0.42 and that the C selected is the smallest one (strongest regularization). In case of tie between cv_scores you should always pick the most regularized model.

Reply:
so larger alpha then

Reply:
Actually, tests are failing. We pick up the smallest alpha instead of the largest one.

Reply:
The reason is that we loop over the `alphas` in the order in the constructor.
Therefore, it will keep `alphas[0]` in all cases. If we want to apply your suggestion we need to change the behaviour of `RidgeCV` and `RidgeClassifierCV`

Reply:
We can address this in another PR then.

Reply:
+1 to do it in another PR

Reply:
Can I work this PR?

Reply:
I already open one there: https://github.com/scikit-learn/scikit-learn/pull/16029
But feel free to comment and contribute. I think that I found some other bugs.